content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md

```diff
@@ -0,0 +1,223 @@
+It got a new CLI and now it is separate from kubectl get. I think that changes in the CLI are always a little bit more apparent because they are user-facing. 
+
+**Are there a lot of other user-facing changes, or are most of the things in the release very much behind the scenes?**<br>
+I would say it's a good mix of both; it depends on what you're interested in. 
+
+**I am interested, of course, in [non-graceful node shutdown support](https://github.com/kubernetes/enhancements/issues/2268). What can you tell us about that?**<br>
+Right, so for situations where you have a hardware failure or a broken OS, we have added additional support for a better graceful shutdown.
+
+**If someone trips over the power cord at your LAN party and your cluster goes offline as a result?**<br>
+Right, exactly. More availability! That's always good. 
+
+**And if it's not someone tripping over your power cord, it's probably DNS that broke your cluster. What's changed in terms of DNS configuration?**<br>
+Oh, we introduced [a new feature gate to allow more DNS search path](https://github.com/kubernetes/enhancements/issues/2595).
+
+**Is that all there is to it?**<br>
+That's pretty much it. [LAUGHING] Yeah, you can have more and longer DNS search path.
+
+**It can never be long enough. Just search everything! If .com doesn't work, try .net and try .io after that.**<br>
+Surely.
+
+**Those are a few of the big features that are moving to stable. Obviously, over the course of the last few releases, features come in, moving from Alpha to Beta and so on. New features coming in today might not be available to people for a while. As you mentioned, there are feature gates that you can enable to allow people to have access to these. What are some of the newest features that have been introduced that are in Alpha, that are particularly interesting to you personally?**<br>
+I have two. The first one is [`kubectl delete --interactive`](https://github.com/kubernetes/enhancements/issues/3895). I'm always nervous when I delete something, you know, it's going to be a typo or it's going to be on the wrong tab. So we have an `--interactive` flag for that now.
+
+**So you can get feedback on what you're about to delete before you do it?**<br>
+Right; confirmation is good!
+
+**You mentioned two there, what was the second one?**<br>
+Right; this one is close to my heart. It is a SIG Release KEP, [publishing on community infrastructure](https://github.com/kubernetes/enhancements/issues/1731). I'm not sure if you know, but as part of my branch management associate role in 1.27, I had the opportunity to cut a few releases. It takes up to 12 hours sometimes. And now, we are hoping that that process only includes release managers, so we don't have to call up the folks at Google and, you know, lengthen that process anymore.
```

Comment:
```suggestion
Right; this one is close to my heart. It is a SIG Release KEP, [publishing on community infrastructure](https://github.com/kubernetes/enhancements/issues/1731). I'm not sure if you know, but as part of my branch management associate role in 1.27, I had the opportunity to cut a few releases. It takes up to 12 hours sometimes. And now, we are hoping that the process only includes release managers, so we don't have to call up the folks at Google and, you know, that won't lengthen the process anymore.
```

Reply:
suggestion is not grammatically correct, and isn't reflective of what was spoken.

(Will take s/that/the/ though, thank you!)